MoltenVK/MoltenVK/Commands/MVKCommandEncodingPool.h

```diff
@@ -151,6 +156,9 @@ class MVKCommandEncodingPool : public MVKBaseObject {
     id<MTLDepthStencilState> _cmdClearDefaultDepthStencilState = nil;
     id<MTLComputePipelineState> _mtlCopyBufferBytesComputePipelineState = nil;
 	id<MTLComputePipelineState> _mtlFillBufferComputePipelineState = nil;
+#if MVK_MACOS
+	id<MTLComputePipelineState> _mtlClearColorImageComputePipelineState[3] = {nil, nil, nil};
+#endif
```

Comment:
@cdavis5e This declaration is guarded with `MVK_MACOS` but not references to this variable in `MVKCommandEncodingPool::destroyMetalResources()` are not. Compilation on iOS fails as a result.

Reply:
Sorry. #1040 should fix it.